From 19e1c5b6dfdec8b328b8e384de3965c166a88203 Mon Sep 17 00:00:00 2001
From: Jean Trivelly <jean.trivelly@intel.com>
Date: Tue, 18 Jun 2013 12:00:51 +0200
Subject: [PORT FROM MAIN] WiFi: Enable HotSpot 2.0 (wpa_supplicant)

BZ: 116962

Enable HotSpot 2.0

Change-Id: Ieed953b97eec09192a0abe23b01fb3e9639cd9a2
Orig-Change-Id: I6112eb40a97980ef685b8b02c58733b4471b0eac
Category: device enablement
Domain: CWS.WIFI-Common
Origin: upstream
Upstream-Candidate: no,proprietary
Signed-off-by: Jean Trivelly <jean.trivelly@intel.com>
---
 wpa_supplicant/Android.mk     |    3 +++
 wpa_supplicant/android.config |    4 ++--
 2 files changed, 5 insertions(+), 2 deletions(-)

diff --git a/wpa_supplicant/Android.mk b/wpa_supplicant/Android.mk
index 880b468..5303a43 100644
--- a/wpa_supplicant/Android.mk
+++ b/wpa_supplicant/Android.mk
@@ -1581,6 +1581,9 @@ include $(BUILD_EXECUTABLE)
 #LOCAL_STATIC_LIBRARIES := libCustomWifi
 #endif
 #LOCAL_SHARED_LIBRARIES := libc libcrypto libssl
+#ifdef CONFIG_PCSC
+#LOCAL_SHARED_LIBRARIES += libpcsclite
+#endif
 #LOCAL_CFLAGS := $(L_CFLAGS)
 #LOCAL_SRC_FILES := $(OBJS_t)
 #LOCAL_C_INCLUDES := $(INCLUDES)
diff --git a/wpa_supplicant/android.config b/wpa_supplicant/android.config
index 771d396..93fe809 100644
--- a/wpa_supplicant/android.config
+++ b/wpa_supplicant/android.config
@@ -490,10 +490,10 @@ CONFIG_IEEE80211N=y
 # This can be used to enable functionality to improve interworking with
 # external networks (GAS/ANQP to learn more about the networks and network
 # selection based on available credentials).
-#CONFIG_INTERWORKING=y
+CONFIG_INTERWORKING=y
 
 # Hotspot 2.0
-#CONFIG_HS20=y
+CONFIG_HS20=y
 
 # Disable roaming in wpa_supplicant
 CONFIG_NO_ROAMING=y
-- 
1.7.4.1

